core/src/avm2/activation.rs

```diff
@@ -1742,6 +1743,30 @@ impl<'a, 'gc> Activation<'a, 'gc> {
         Ok(FrameControl::Continue)
     }
 
+    fn op_find_def(
+        &mut self,
+        method: Gc<'gc, BytecodeMethod<'gc>>,
+        index: Index<AbcMultiname>,
+    ) -> Result<FrameControl<'gc>, Error<'gc>> {
+        let multiname = self.pool_multiname_static(method, index)?;
+        avm_debug!(self.avm2(), "Resolving {:?}", *multiname);
+        if let Some((_, mut found)) = self.domain().get_defining_script(&multiname)? {
+            let object = found.globals(&mut self.context)?;
+            self.push_stack(object);
+        } else {
+            return Err(Error::AvmError(crate::avm2::error::reference_error(
```

Comment:
I think the impl should be next to Domain::get_defined_value, as most of the code there is nearly identical
(in fact, get_defined_value could use this new method under the hood, right? It's just find_def + get_property)